README.md

```
@@ -1,1152 +1,367 @@
-- `POST /auth/token` - トークン生成
-- `GET /health`, `/ready`, `/live` - ヘルスチェック
-- `GET /metrics` - Prometheusメトリクス
-- `GET /swagger/*` - API ドキュメント
+### アクセス
 
-**破壊的変更** (v2.0):
-- **v1.x**: GETリクエストは認証不要
-- **v2.0**: GETリクエストを含むすべてのメソッドで保護エンドポイントへのアクセスにJWT認証が必須
+| サービス | URL | 説明 |
+|---------|-----|------|
+| API | http://localhost:8080 | REST API |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)